src/server/ApiRoutes/Account.js

```diff
@@ -107,8 +108,13 @@ function postToAccountPage(req, res) {
     .catch(resp => res.json({ error: resp }));
 }
 
+function logError(req) {
+  logger.error('Account Error', req.url);
```

Comment:
Since this is an open ajax endpoint, we should sanitize to ensure no active URLs get inserted in our logs. Minimally (to remove the protocol, for example "https://"):
```
  logger.error('Account Error', req.url.replace(/\w+:\/\//g, ''));
```

Reply:
Fixed.